src/main/java/com/bmc/truesight/saas/remedy/integration/impl/GenericTemplateValidator.java

```diff
@@ -118,4 +133,12 @@ public boolean validate(Template template) throws ValidationException {
         return true;
     }
 
+    private boolean isValidAppId(String inputString) {
```

Comment:
Please move this method into Util class

Reply:
As discussed, this is very specific to validation & not a generic utility, so keeping it here seems ok.